### PR TITLE
Feature book sporfield

### PR DESCRIPTION
**Pull request by condef5**
### Status
Required

### Implementation Notes

* Add calendar for bookings in sport fields
* Selected date and hour in the calendar
* Pass info to redux.

### CODE REVIEW

File: client/src/components/calendar.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  function back() {
+    if (
+      dateFns.differenceInCalendarDays(
+        cursor,
+        dateFns.startOfWeek(new Date())
+      ) >= 7
+    ) {
+      setCursor(dateFns.subDays(cursor, 7));
+    } else {
+      alert("Please select an available booking date");
```
Comment (liannivin) [inline client/src/components/calendar.js]: It seems as a practical way to validate.

File: client/package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "@reach/router": "^1.2.1",
     "@testing-library/react": "^8.0.1",
     "geolib": "^3.0.4",
+    "date-fns": "^1.30.1",
```
Comment (cristianbgp) [inline client/package.json]: What is this package for?

File: client/src/components/calendar.js (hunk 1 above)
Comment (condef5) [reply client/src/components/calendar.js]: Thanks, man `:)`, we will always carry you in our hearts

File: client/src/services/club.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ async function unfavorite(clubId) {
   return response.json();
 }
 
-async function clubData(id) {
+async function getClub(id) {
```
Comment (cristianbgp) [inline client/src/services/club.js]: Thanks, that is a better name

File: client/package.json (hunk 2 above)
Comment (condef5) [reply client/package.json]: This package is for to work with dates, `date-fns` is an awesome library, you can read more [here](https://date-fns.org/).

**Pull request merged**